gc_apps/gis_tabular/views_delete.py

```diff
@@ -104,3 +108,85 @@ def view_delete_tabular_map(request):
     d['page_title'] = PANEL_TITLE_REMAP
 
     return render(request, 'worldmap_layers/view_delete_layer.html', d)
+
+def view_delete_tabular_map_no_ui(request, dataverse_token):
+    """
+    Attempt to delete a dataverse-created WorldMap layer via a direct API call, w/out any UI involved.
+    Operates on the same token-callback url principle as other geoconnect methods.
+    (1) Check incoming url for a callback key 'cb'
+        and use the callback url to retrieve the DataverseInfo via a POST
+        (this is copied directly from the workflow of the "map-it" method)
+
+        OK, the whole process has been royally simplified! We are essentially 
+        dropping the previously performed steps 2-4. It is in fact possible to 
+        make a WorldMap API delete layer call *by the datafile_id and the 
+        dataverse installation name*. And we already have these in the dict 
+        that we received from following the callback URL. 
+    (2) [SKIP] If the Datafile info looks good, download and process the datafile
+        (yes, we do have to actually download the file, even if we are merely deleting a map layer... 
+        unless I'm missing something)
+    (3) [SKIP] If that worked, look up TabularInfo for this file, using the md5 produced in step (2)
+    (4) [SKIP] Look up MapLayerInfo by the TabularInfo produced in step (3)
+    (5) Try deleting the map layer on the WorldMap side
+        (a new call has been added to the layer_services, that takes a dict, containing 
+        the datafile_id and dataverse_installation_name
+    (6) And if that worked too, delete it locally, in GeoConnect.
+    (7) Return 200 and a success message.
+    """
+
+    # Process the incoming url for a callback key 'cb'
+    # and use the callback url to retrieve the DataverseInfo via a POST
+    # (this is copied directly from the workflow of the "map-it" method)
+    request_helper = InitialRequestHelper(request, dataverse_token)
+    if request_helper.has_err:
+        return HttpResponse(MessageHelperJSON.get_json_fail_msg("Failed to obtain datafile metadata from the Dataverse. Error: " + request_helper.err_msg), status=412)
+
+    # check that it's a tabular file:
+    if not request_helper.mapping_type == 'tabular':
```

Comment:
Instead of  ```== 'tabular'```, use the static variable "DV_MAP_TYPE_TABULAR", e.g. ```== DV_MAP_TYPE_TABULAR```
```
from gc_apps.layer_types.static_vals import DV_MAP_TYPE_TABULAR
```